modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,65 @@
+= XDCR enableCrossClusterVersioning
+:description: pass:q[Enabling the bucket property `enableCrossClusterVersioning` allows XDCR to add metadata to each replicated document.]
```

Comment:
Since we use `enableCrossClusterVersioning` in the next paragrah, as well as explaining that it's a bucket property, here, instead of:
Enabling the bucket property `enableCrossClusterVersioning` allows XDCR to add metadata to each replicated document.
Let's say:
Enabling Cross Cluster Versioning allows XDCR to add metadata to each replicated document.